doc(release): prepare release notes for oss 23.04 (#2505)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx

## Centreon Web

### 23.04.6

> Release date: `soon`

Reviewer: ```suggestion
Release date: `2023-07-28`
```

<details open>
  <summary>Bug fixes</summary>

> - [Core] Removed non necessary data in custom_configuration column for SAML provider.

Reviewer 2: ```suggestion
- [Core] Removed unnecessary data in custom_configuration column for SAML provider.
```

- [Install] Updated size to metric_name column to 1021.

> - [Resources Status] Fixed CSV export when metric name contains SQL keyword.

Reviewer 2: ```suggestion
- [Resources Status] Fixed CSV export when metric name contained SQL keyword.
```

## Centreon Gorgone

### 23.04.4

> Release date: `soon`

Reviewer: ```suggestion
Release date: `July 28, 2023`
```

<details>
  <summary>Bug fixes</summary>

- [Packaging] Improved packaging by adding gorgone_key_thumbprint.pl script and perl-Mojolicious dependency.

> </details/>

Reviewer 2: ```suggestion
</details>
```

## Centreon Open Tickets

### 23.04.2

> Release date: `soon`

Reviewer: ```suggestion
Release date: `July 28, 2023`
```

<details open>
  <summary>Bug fixes</summary>

> - Fixed a broker query.

Reviewer 2: ```suggestion
- Fixed a Broker query.
```
